File: automacao/programacao-frc/exemplo-joystick.java
import edu.wpi.first.wpilibj.Joystick;

public class Robot extends TimedRobot {
	private final Joystick m_stick = new Joystick(0);

	@Override
	public void teleopPeriodic() {
		if (m_stick.getRawButton(0)) {
			// Botão apertado, faça algo.
		} else {
			// Botão não pressionado, faça outra coisa.
		}
	}
}

File: automacao/programacao-frc/exemplo-joystick.cpp
#include <frc/Joystick.h>

class Robot : public frc::TimedRobot
{
	frc::Joystick m_stick{0};

public:
	void TeleopPeriodic()
	{
		if (m_stick.GetRawButton(0))
		{
			// Botão apertado, faça algo.
		}
		else
		{
			// Botão não pressionado, faça outra coisa.
		}
	}
};

#ifndef RUNNING_FRC_TESTS
int main()
{
	return frc::StartRobot<Robot>();
}
#endif
